examples/integrations/cdi/datasource-hikaricp-mysql/README.adoc

```diff
@@ -0,0 +1,95 @@
+= MySQL Integration Example
+
+== Overview
+
+This example shows a trivial Helidon MicroProfile application that
+uses the MySQL CDI integration, how to package it into a Docker image,
+and how to run it in a Docker container.  It also shows how to run
+MySQL in a Docker container and connect to it using the application's
+container.
+
+== Notes
+
+To create a user-defined Docker network:
```

Comment:
Is this required? Or are you doing this to make the example a little more realistic? No problem either way.

Reply:
Unless it is required I'd say we should remove as it adds more complexity.

Reply:
This is just so that the end-to-end-ness of this example is realized in as brief a way as possible.  Otherwise I don't know what the IP is (for example) of the MySQL Docker container.

Reply:
We could choose to not run the demo app in a container and just use localhost with a port fixed binding. Otherwise maybe `docker run --link` can be used ?

Reply:
User networks are the replacement for `--link`.

Reply:
you mean --link is being deprecated ?

Reply:
Anyway, I would favor using docker to run mysql and `mvn exec:java` or `java -jar` to run the application.

Reply:
Yep; see https://docs.docker.com/network/links/.